jenkins-scripts/docker/ign_utils-compilation.bash

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -x
+
+# Knowing Script dir beware of symlink
+[[ -L ${0} ]] && SCRIPT_DIR=$(readlink ${0}) || SCRIPT_DIR=${0}
+SCRIPT_DIR="${SCRIPT_DIR%/*}"
+
+if [[ -z ${ARCH} ]]; then
+  echo "ARCH variable not set!"
+  exit 1
+fi
+
+if [[ -z ${DISTRO} ]]; then
+  echo "DISTRO variable not set!"
+  exit 1
+fi
+
+export BUILDING_SOFTWARE_DIRECTORY="ign-utils"
+export BUILDING_PKG_DEPENDENCIES_VAR_NAME="IGN_UTILS_DEPENDENCIES"
```

Comment:
There is definition of `IGN_UTILS_DEPENDENCIES`. Should go in the dependencies_archive together with the other deps.

Reply:
271118f